packages/core/src/components/split-button/split-button.tsx

```diff
@@ -114,11 +124,18 @@ export class SplitButton {
       variant: this.variant,
       outline: this.outline,
       ghost: this.ghost,
-      disabled: this.disabled,
+      disabled: this.disabled || this.isDisabledButton,
       class: {
         'left-button-border': !this.outline,
       },
     };
+    const disabledButtonAttributes = {
+      variant: this.variant,
+      outline: this.outline,
+      ghost: this.ghost,
+      disabled: this.disabled || this.isDisabledIcon,
+      class: { anchor: true },
+    };
```

Comment:
Some points regarding this change:

- Object name is misleading.
- variant/outline/ghost are part of the overall button props does not need to be handled again here
-Why is there a class anchor introduced, which is already part of the tsx element itself